msi/GitProduct.wxs

```diff
@@ -284,6 +286,9 @@
       <Custom Action="ExecConfigureCoreAutoCrlf" After="ExecPostInstallBat">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
+      <Custom Action="ExecConfigureSslCAInfo" After="ExecConfigureCoreAutoCrlf">
+        <![CDATA[REMOVE <> "ALL"]]>
+      </Custom>
```

Comment:
This ensures that this custom action is not executed on uninstall. No need to set this git configuration setting when uninstalling Git for Windows.